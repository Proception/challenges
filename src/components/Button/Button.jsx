import React, { Component } from 'react';

const Button = (props) => {
  return (
    <button className={props.classButton} disabled={props.disabled}
      id={props.id}
      onClick={ props.disabled ? '' : props.action}>
      <div className= {props.classTitle}>
        {props.title} 
      </div>
    </button>
  );
};

export default Button;
