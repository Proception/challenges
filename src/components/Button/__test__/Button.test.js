import React from 'react';
import renderer from 'react-test-renderer';
import Button from '../Button.jsx';


test('Button component', () => {
  const component = renderer.create(
    <Button />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
