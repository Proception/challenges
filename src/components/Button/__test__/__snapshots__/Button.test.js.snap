// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component 1`] = `
<button>
  <div />
</button>
`;
